posthog/types.py

```diff
@@ -3,6 +3,7 @@
 from posthog.models.filters.filter import Filter
 from posthog.models.filters.path_filter import PathFilter
 from posthog.models.filters.retention_filter import RetentionFilter
+from posthog.models.filters.session_recordings_filter import SessionRecordingsFilter
 from posthog.models.filters.stickiness_filter import StickinessFilter
 
-FilterType = Union[Filter, PathFilter, RetentionFilter, StickinessFilter]
+FilterType = Union[Filter, PathFilter, RetentionFilter, StickinessFilter, SessionRecordingsFilter]
```

Comment:
This shouldn't be in this PR right?

This "implicitly" makes column_optimizer support session recordings filters which isn't the case and is scary as heck.